opte-api/src/lib.rs

```diff
@@ -58,7 +58,7 @@ pub use ulp::*;
 ///
 /// We rely on CI and the check-api-version.sh script to verify that
 /// this number is incremented anytime the oxide-api code changes.
-pub const API_VERSION: u64 = 22;
+pub const API_VERSION: u64 = 23;
```

Comment:
Is this PR actually changing the API as perceived by ioctl consumers?

Reply:
It is, yeah. There's a new error variant, `OpteError::InvalidIpCfg`, which is returned if there is an external IP address, but it has zero ports.